zephyr/esp32/CMakeLists.txt

```diff
@@ -11,8 +11,8 @@ if(CONFIG_SOC_SERIES_ESP32 OR CONFIG_SOC_SERIES_ESP32_NET)
     ../esp_shared/include
     ../esp_shared/include/wifi
     ../esp_shared/components/include
-    ../../components/hal/include
     ../../components/hal/esp32/include
+    ../../components/hal/include
```

Comment:
You could restore this line as it was not modified.

Reply:
The order of the lines 14 a 15 matters because inside ../../components/hal/esp32/include there is the following preprocessor instruction:
#include_next "hal/touch_sensor_hal.h"